packages/manager/apps/pci-quota/src/pages/regions/Regions.page.tsx

```diff
@@ -0,0 +1,169 @@
+import { OdsButton, OdsMessage, OdsText } from '@ovhcloud/ods-components/react';
+import { useEffect, useState } from 'react';
+import { Translation, useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+import {
+  Notifications,
+  useNotifications,
+} from '@ovh-ux/manager-react-components';
+import {
+  isDiscoveryProject,
+  PciAnnouncementBanner,
+  PciDiscoveryBanner,
+  useProject,
+} from '@ovh-ux/manager-pci-common';
+import { useMedia } from 'react-use';
+import { AvailablePart } from '@/pages/regions/Available.part';
+import { ToAddPart } from '@/pages/regions/ToAdd.part';
+import { useGetAvailableRegions, useLocations } from '@/api/hooks/useRegions';
+import { addRegion } from '@/api/data/region';
+import queryClient from '@/queryClient';
+
+export type TPlainLocation = {
```

Comment:
Does this adds something more than other types with the same name ?

Reply:
looks like it is used to type the region mapping returned by the hooks